tests/functional/UseCase/Bucket/GrantRevokeBucketAccessToReadOnlyRoleTest.php

```diff
@@ -215,6 +213,61 @@ public function testRegisterBucket(): void
         }
     }
 
+    public function testRegisterBucketInDifferentRegionShouldFail(): void
+    {
+        $externalTableName = md5($this->getName()) . '_Test_table';
```

Comment:
proč nepoužíváš `$this->getTestHash()`? o řádek níž jo.

Reply:
opravene dik, pridal som este logs do ignoru